_presenters/eric-holscher.md

```diff
@@ -4,10 +4,10 @@ hidden: false
 layout: speaker-template
 name: Eric Holscher
 permalink: /presenters/eric-holscher/
-photo_url: null
+photo_url: https://pbs.twimg.com/profile_images/686618084279336960/9UXiuf6J_400x400.jpg
```

Comment:
We do prefer hosting them ourselves just to avoid being at the mercy of third-party services (any more than we have to be at least). I can upload it or you can. Either way is fine by me.